CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 * [ENHANCEMENT] Add length and limit to labelNameTooLongError and labelValueTooLongError #4595
 * [ENHANCEMENT] Add jitter to rejoinInterval. #4747
 * [ENHANCEMENT] Compactor: uploading blocks no compaction marks to the global location and introduce a new metric #4729
+* [ENHANCEMENT] Logger: add possibility to implement `PrometheusLoggerFactory` to InitLogger #4786
```

Comment:
Lets move this to the master releases.
` master / unreleased`

The 1.13.0 already started